--- a/fftools/ffmpeg.c
+++ b/fftools/ffmpeg.c
@@ -141,6 +141,11 @@
    This is a temporary solution until libavfilter gets real subtitles support.
  */
 
+__attribute__((visibility("default")))
+int ffmpeg_main(int argc, char **argv);
+
+int main(int argc, char **argv);
+
 static void sub2video_heartbeat(InputFile *infile, int64_t pts, AVRational tb)
 {
     /* When a frame is read from a file, examine all sub2video streams in
@@ -1350,3 +1355,9 @@
     ffmpeg_cleanup(ret);
     return ret;
 }
+
+__attribute__((visibility("default")))
+int ffmpeg_main(int argc, char **argv)
+{
+    return main(argc, argv);
+}
